src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -166,5 +166,12 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// </remarks>
         /// </summary>
         public bool UseDefaultJsonSerialization { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Gets or sets the DateTime from which feed should start from. 
+        /// This is useful when you need to skip past and start from a checkpoint. It is best to use UTC date / time format to avoid any discrepencies.
```

Comment:
We need to add validation, what happens if the user sets StartFrom and StartFromBeginning at the same time?

Reply:
There should be no conflict. StartFromBeginning has no effect when leases are specified and created.

Reply:
StartFromTime also has no effect if leases are created, it only is useful to define the first state. Same as StartFromBeginning.

The validation is for the user's sake, if they set StartFromBeginning = true, and StartTime = "XX", which is the start point? It is not clear for the user, so we need to avoid combinations that can lead to an undefined behavior.

Reply:
updated